### PR TITLE
[ZEPPELIN-2179] "clear output" paragraph doesn't work in personalized mode (branch-0.7)

### PR DESCRIPTION
**Pull request by 1ambda**
### What is this PR for?

`clear output` (`cmd` + `opt` + `L`) doesn't work in the personalized mode.

- CI failure might be related with https://github.com/apache/zeppelin/pull/2103

### What type of PR is it?
[Bug Fix]

### Todos

NONE

### What is the Jira issue?

[ZEPPELIN-2179](https://issues.apache.org/jira/browse/ZEPPELIN-2179)

### How should this be tested?

1. Execute a paragraph
2. Click `clear output` button (`CMD` + `OPT` + `L)

### Screenshots (if appropriate)

NONE

### Questions:
* Does the licenses files need update? - NO
* Is there breaking changes for older versions? - NO
* Does this needs documentation? - NO

### CODE REVIEW

**Comment by 1ambda** [comment]
@jongyoul Can you help review this?

File: zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java [hunk 1]
```diff
@@ -628,7 +628,7 @@ public void broadcastParagraph(Note note, Paragraph p) {
   }
 
   public void broadcastParagraphs(Map<String, Paragraph> userParagraphMap,
-      Paragraph defaultParagraph) {
+                                  Paragraph defaultParagraph) {
```
Comment (jongyoul) [inline zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java]: @1ambda Please turn off this setting of vertical alignment.

Comment (1ambda) [inline zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java]: @jongyoul I'v just removed the `defaultParagraph` parameter which is not used at all in b1e782e

**Comment by 1ambda** [comment]
@jongyoul If implementation is ok, I will create a PR for the master branch as well.

**Comment by Leemoonsoo** [comment]
CI failure on `branch-0.7` is fixed. @1ambda Could you rebase this PR to `branch-0.7` and see if CI passes?

**Comment by 1ambda** [comment]
I'v just rebased, let me comment after fixing CI.

**Comment by 1ambda** [comment]
CI passed. @Leemoonsoo

**Comment by 1ambda** [comment]
Regarding to the same fix for master branch, I will open another PR. 
Thanks for reviewing this PR @Leemoonsoo @jongyoul!

**Comment by jongyoul** [comment]
@1ambda I merged this PR into branch-0.7. Please close this PR and make a new PR for master. I've tried to merge it into master but ,as you mentioned, it wouldn't be merged because of conflict.

**Comment by 1ambda** [comment]
Thanks @jongyoul!

**Pull request closed without merging**